src/main/scala/census/db/creator/service/actors/UnzipActor.scala

```diff
@@ -0,0 +1,45 @@
+package census.db.creator.service.actors
```

Comment:
I'd suggest simplifying this by using https://github.com/srikanth-lingala/zip4j

```
new ZipFile("filename.zip").extractAll("/destination_directory");
````